build-system-tests/scripts/checkReactNativeLog.ts

```diff
@@ -91,7 +91,10 @@ const checkStartMessage = async (
  * @returns {boolean} hasError
  */
 const checkErrorMessage = async (logLines: string[]): Promise<boolean> => {
-  log('info', `Checking log file ${logFileName} for errors...`);
+  log(
+    'info',
+    `Checking log file ${logFileName} on  ${platform} platform for errors...`
```

Comment:
In a previous commit I removed the usage of the platform var (it was used to have a slower timeout on ios than android, but ios tests were flaky so using same timeout now). Didn't want to entirely remove the platform var since it may come in handy in the future. Using it in log for now.